lms/templates/instructor/edx_ace/accountcreationandenrollment/email/body.html

```diff
@@ -0,0 +1,41 @@
+{% extends 'ace_common/edx_ace/common/base_body.html' %}
+
+{% load i18n %}
+{% load static %}
+{% block content %}
+<table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
+    <tr>
+        <td>
+            <h1>
+                {% blocktrans %}
+                    Welcome to {{ course_name }}
+                {% endblocktrans %}
+            </h1>
+
+            <p style="color: rgba(0,0,0,.75);">
+                {% blocktrans %}To get started, please visit https://{{ site_name }}.{% endblocktrans %}
+                <br />
+            </p>
+
+            <p style="color: rgba(0,0,0,.75);">
+                {% blocktrans %}The login information for your account follows:{% endblocktrans %}
+                <br />
+                {% blocktrans %}email: {{ email_address }}{% endblocktrans %}
+                <br />
+                {% blocktrans %}password: {{ password }}{% endblocktrans %}
```

Comment:
Really? there's a template that sends passwords in plaintext emails? 😱

Reply:
Yeah, I guess :slightly_frowning_face: